fixed external OAuth documentation code snippet comments (#2411)
TykTechnologies/tyk-docs: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md

      notBeforeValidationSkew: 0
      expiresAtValidationSkew: 0
      identityBaseField: # identity claimName
    introspection: # array for introspection details
      enabled: true/false
      clientID: # for introspection request
      clientSecret: # for introspection request, if empty will use oAuth.secret
      url: # token introspection endpoint

>       cache: # Tyk caches the introspection response when enabled is set true inside the cache configuration of introspection

Reviewer: ```suggestion
      cache: # Tyk will cache the introspection response when `cache.enabled` is set to `true`
```

        enabled: true/false,

>         timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)

Reviewer: I wondered that! @lghiur?

>         timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)

Reviewer: It means that it is cached forever

>         timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)

Reviewer: ```suggestion
        timeout: 0 # Length of time for which Tyk should cache the introspection result; if set to zero (default) then cache does not expire (seconds)
```

>         timeout: 0 # Tyk caches the introspection result until the timeout runs out. The value is in seconds

Reviewer: ```suggestion
        timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)
```

Reviewer 2: what does it mean zero timeout if the cache is enabled?

>         timeout: 60 # Tyk caches the introspection result until the timeout runs out. The value is in seconds

Reviewer 2: is 60 the default?

>         timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds). "0" means that the introspection result is cached by the end of life of the token

Reviewer 2: ```suggestion
        timeout: 0 # The duration (in seconds) for which Tyk will retain the introspection outcome in its cache. If the value is "0", it indicates that the introspection outcome will be stored in the cache until the token's expiration.
```
